src/main/java/mate/controllers/driver/DeleteDriverController.java

```diff
@@ -0,0 +1,25 @@
+package mate.controllers.driver;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/delete/")
```

Comment:
```suggestion
@WebServlet(urlPatterns = "/drivers/delete")
```
url patterns should be consistent between each other 
if you have /drivers/add pattern without last slash in another controller, so in this URL you should not have it too